### PR TITLE
Fix en leaks bugs

### PR DESCRIPTION
**Pull request by nikhilvenkatraman-okta**
Resolves: OKTA-388796

## Description:

Fixes few more en leaks bugs 

## PR Checklist

- [x] Have you verified the basic functionality for this change?

- [x] Added e2e tests

### Screenshot/Video:

Screen Shot 2021-04-20 at 2 47 01 PM

### Reviewers:

@jmelberg-okta @haishengwu-okta 

### Issue:

- [OKTA-388796](https://oktainc.atlassian.net/browse/OKTA-388796)

### CODE REVIEW

File: playground/main.js [hunk 1]
```diff
@@ -88,8 +88,14 @@ const renderPlaygroundWidget = (options = {}) => {
       // wait for view to finish rendering
       const viewText = document.getElementById('okta-sign-in').textContent;
       const noTranslationContentExists = document.getElementsByClassName('no-translate').length;
-      const noTranslationContent = noTranslationContentExists &&
-        document.getElementsByClassName('no-translate')[0].textContent;
+      let noTranslationContent = '';
+      /* eslint max-depth: [2, 3] */
+      if (noTranslationContentExists) {
+        const noTranslateElems = document.getElementsByClassName('no-translate');
```
Comment (nikhilvenkatraman-okta) [inline playground/main.js:94]: Handle multiple elements with class of `no-translate`

File: test/testcafe/spec-en-leaks/EnglishLeaks_spec.js [hunk 2]
```diff
@@ -164,7 +158,13 @@ const testEnglishLeaks = (mockIdxResponse, fileName, locale) => {
     const viewTextExists = await Selector('#okta-sign-in').exists;
     const viewText = viewTextExists && await Selector('#okta-sign-in').textContent;
     const noTranslationContentExists = await Selector('.no-translate').exists;
-    const noTranslationContent = noTranslationContentExists && await Selector('.no-translate').textContent;
+    let noTranslationContent = '';
```
Comment (nikhilvenkatraman-okta) [inline test/testcafe/spec-en-leaks/EnglishLeaks_spec.js:161]: Handle multiple elements with class of `no-translate`

**Review by haishengwu-okta — APPROVED** [review]

File: src/v2/view-builder/views/ov/NumberChallengePhoneView.js [hunk 3]
```diff
@@ -4,14 +4,14 @@ import hbs from 'handlebars-inline-precompile';
 const NumberChallengePhoneView = View.extend({
   className: 'number-challenge-section',
   template: hbs`
-     <p>
+     <p class="no-translate">
```
Comment (jmelberg-okta) [inline src/v2/view-builder/views/ov/NumberChallengePhoneView.js:7]: Why is this marked as "no-translate" given there is an `i18n` message below?

Comment (nikhilvenkatraman-okta) [reply src/v2/view-builder/views/ov/NumberChallengePhoneView.js:7]: Good catch, I am fixing this in my next commit. I just realized that there is a bug in the replacement logic where we are not replacing every instance of the noTranslate content.

The current logic only replaces the first instance of a noTranslationContent.
```
viewText = viewText.replace(noTranslationContent, '');
```

Comment (nikhilvenkatraman-okta) [reply src/v2/view-builder/views/ov/NumberChallengePhoneView.js]: Screen Shot 2021-04-20 at 3 31 42 PM

For more context, this is the view where we want to ignore number 10 in 2 places

**Review by jmelberg-okta — APPROVED** [review]

**Pull request closed without merging**